### ISSUE TITLE
Built-in `min` and `max` missing for f16{, le, be}

### ISSUE THREAD

**Issue by Kelimion** [odin-lang/Odin#1004]
## Context

* Operating System: Windows 10
* Please paste `odin version` output: odin version dev-2021-05:c8b353b6

## Expected Behavior

baz = foo;

## Current Behavior

```
odin run t.odin
t.obj : error LNK2019: unresolved external symbol runtime.min_f16 referenced in function minrep.main
W:\odin-test\t.exe : fatal error LNK1120: 1 unresolved externals
```

### Steps to Reproduce

```odin
package minrep

main :: proc() {

	foo: f16 = 1.0;
	bar: f16le = 2.0;

	baz := min(f16le(foo), bar);
}
```

**Comment by Kelimion** [comment]
`abs` is missing for `f16` as well and as a result of `min` and `max` being missing, `clamp` also fails on `f16`.

**Comment by Kelimion** [comment]
Partially fixed by #1005.
